packages/caip/src/adapters/coinbase/getCoinbasePayAssets.ts

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios'
+import toLower from 'lodash/toLower'
+
+import { AssetId } from '../../index'
+
+/** Helper module to generate the coinbase supported assets map in the sibling index file */
```

Comment:
How are we using this, is it supposed to be used as a CLI of sorts? If so, we can source `assetIdToCoinbaseAssetIdMap` from a JSON file instead and `fs.writeFile` it or similar

Reply:
Originally, I added this code because I had it in web and I didn't want to lose the functions used to generate the object.

I decided to add a generate script around this module and wrote the output into a json file that is used as the map.